### PR TITLE
vulkan: Add version 1.2.135.0

### PR DESCRIPTION
**Pull request by sinloss**
Introduce the `vulkan sdk` to this `main` bucket.

As there are many types of licenses involved, I cannot determine a proper `identifier` of license, it would be great if someone could help me with that...

### CODE REVIEW

**Comment by linsui** [comment]
> The Vulkan SDK is comprised of 100% open source components. The majority of the materials are MIT or Apache 2.0 licenses. This Vulkan SDK license registry discloses all components in the SDK and their corresponding open source license and copyrights.

I suggest 
```
"license": {
    "identifier": "MIT,Apache-2.0",
    "url": "https://vulkan.lunarg.com/license/"
},
```

File: bucket/vulkan.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        "hash": {
+            "url": "https://vulkan.lunarg.com/sdk/home.dhtml",
+            "regex": ">([\\da-f]{64})<"
+        }
+    },
+    "notes": [
+        "Please make sure you have the vulkan driver installed.",
+        "Variety of drivers of different graphic card vendors could be",
+        "found at the bottom of https://www.khronos.org/vulkan/"
+    ]
```
Comment (Ash258) [inline bucket/vulkan.json:41]:
```suggestion
    }
```

File: bucket/vulkan.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+    "version": "1.2.135.0",
+    "description": "Vulkan is a new generation graphics and compute API that provides high-efficiency, cross-platform access to modern GPUs used in a wide variety of devices from PCs and consoles to mobile phones and embedded platforms.",
+    "homepage": "https://www.khronos.org/vulkan/",
+    "license": {
+        "identifier": "Multiple Licenses"
+    },
```
Comment (Ash258) [inline bucket/vulkan.json]:
```suggestion
    "description": "SDK for new generation graphics and compute API",
    "homepage": "https://www.khronos.org/vulkan/",
    "license": {
        "identifier": "MIT,...",
        "url": "https://vulkan.lunarg.com/license/#/license"
    },
    "notes": [
        "Make sure you have the vulkan driver installed.",
        "Variety of drivers of different graphic card vendors could be found at the bottom of 'https://www.khronos.org/vulkan/'"
    ],
```

File: bucket/vulkan.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "url": "https://sdk.lunarg.com/sdk/download/1.2.135.0/windows/VulkanSDK-1.2.135.0-Installer.exe#/dl.7z",
+    "hash": "181b3353612c8b0fc5e1857b652d62140191ae42b98f2f0d532cf349cebfd8c6",
+    "pre_install": "Remove-Item -R -Path \"$dir\\`$TEMP\",\"$dir\\`$PLUGINSDIR\",\"$dir\\`$1\",\"$dir\\Uninstall.exe.nsis\"",
```
Comment (Ash258) [inline bucket/vulkan.json:11]:
```suggestion
    "pre_install": "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstal*\" -Recurse",
```

File: bucket/vulkan.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        "32bit": {
+            "env_add_path": [
+                "Bin32",
+                "Tools32"
+            ]
+        }
+    },
+    "checkver": {
+        "url": "https://vulkan.lunarg.com/sdk/home.dhtml",
+        "regex": "#sdk/downloadConfirm/([\\d\\.]+)"
```
Comment (Ash258) [inline bucket/vulkan.json:28]:
```suggestion
        "regex": "/VulkanSDK-([\\d.]+)-Installer\\.exe"
```

File: bucket/vulkan.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    "autoupdate": {
+        "url": "https://sdk.lunarg.com/sdk/download/$version/windows/VulkanSDK-$version-Installer.exe#/dl.7z",
+        "hash": {
+            "url": "https://vulkan.lunarg.com/sdk/home.dhtml",
+            "regex": ">([\\da-f]{64})<"
```
Comment (Ash258) [inline bucket/vulkan.json:34]:
```suggestion
            "regex": "(?sm)$basename.*?>$sha256</"
```

**Comment by sinloss** [comment]
> "license": {
>     "identifier": "MIT,Apache-2.0",
>     "url": "https://vulkan.lunarg.com/license/"
> },

Thanks! 😄

File: bucket/vulkan.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
+{
+    "version": "1.2.135.0",
+    "description": "Vulkan is a new generation graphics and compute API that provides high-efficiency, cross-platform access to modern GPUs used in a wide variety of devices from PCs and consoles to mobile phones and embedded platforms.",
+    "homepage": "https://www.khronos.org/vulkan/",
+    "license": {
+        "identifier": "MIT,Apache-2.0",
+        "url": "https://vulkan.lunarg.com/license/"
+    },
```
Comment (Ash258) [inline bucket/vulkan.json:8]:
```suggestion
    "description": "SDK for new generation graphics and compute API",
    "homepage": "https://www.khronos.org/vulkan/",
    "license": {
        "identifier": "MIT,...",
        "url": "https://vulkan.lunarg.com/license/#/license"
    },
    "notes": [
        "Make sure you have the vulkan driver installed.",
        "Variety of drivers of different graphic card vendors could be found at the bottom of 'https://www.khronos.org/vulkan/'"
    ],
```

**Pull request merged**